src/quacc/recipes/espresso/phonons.py

```diff
@@ -102,7 +104,7 @@ def q2r_job(
 
     Parameters
     ----------
-    prev_dir
+    copy_files
         Outdir of the previously ran ph.x calculation. This is used to copy
         the the dynamical matrix files.
```

Comment:
The docstring no longer matches the parameter. Before, it was a previous directory. Now, it can be many things apparently.